src/App.js

```diff
@@ -59,25 +62,31 @@ const mapStateToProps = (state) => {
 
 class Root extends React.Component {
   componentDidMount() {
-    WebSocketHandler.on('wallet', this.handleWebsocket);
-    WebSocketHandler.on('storage', this.handleWebsocketStorage);
+    hathorLib.WebSocketHandler.on('wallet', this.handleWebsocket);
+    hathorLib.WebSocketHandler.on('storage', this.handleWebsocketStorage);
+
+    hathorLib.axios.registerNewCreateRequestInstance(createRequestInstance);
+
+    hathorLib.WebSocketHandler.on('addresses_loaded', this.addressesLoadedUpdate);
+    hathorLib.WebSocketHandler.on('is_online', this.isOnlineUpdate);
+    hathorLib.WebSocketHandler.on('reload_data', this.reloadData);
   }
 
   componentWillUnmount() {
-    WebSocketHandler.removeListener('wallet', this.handleWebsocket);
-    WebSocketHandler.removeListener('storage', this.handleWebsocketStorage);
+    hathorLib.WebSocketHandler.removeListener('wallet', this.handleWebsocket);
+    hathorLib.WebSocketHandler.removeListener('storage', this.handleWebsocketStorage);
```

Comment:
Shouldn't we remove listener for `address_loaded`, `is_online`, and `reload_data` as well?